week3/homework/src/index.js

```diff
@@ -1,3 +1,55 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+const express = require('express');
+const app = express();
+const add = require('./add');
+const update = require('./update');
+const remove = require('./remove');
+const reset = require('./reset');
+const readTodo = require('./readTodo');
+const markAsDone = require('./markAsDone');
+const markAsNotDone = require('./markAsNotDone');
+
+app.use(express.json());
+
+try {
+  app.get('/todos/:id', (request, response) => {
```

Comment:
there should  also be a get todos that returns a list of all todos

Reply:
i ve made this changes